src/components/attendee/PortalModal/PortalModal.module.scss

```diff
@@ -0,0 +1,36 @@
+@use "scss/attendee/font";
+@use "scss/attendee/effects/atom";
+@use "scss/attendee/effects/rounded";
+@use "scss/attendee/space";
+@use "scss/attendee/effects/sparkle";
+@use "scss/attendee/theme";
+
+.PortalPopupInfo {
+  @include sparkle.panel--glass;
+  @include rounded.normal;
+  border: 1px solid theme.$sparkle--border;
+  box-sizing: border-box;
+  z-index: 100;
+  max-width: 200px;
+  display: flex;
+  flex-direction: column;
+
+  h3 {
```

Comment:
[NOTE]

I'm not sure we should use headings, they have semantic meaning that is hard to track (at least at this stage).
A simple div with maybe a class should be enough to have the correct style, but not have headings that might confuse screen readers.

This should all be revisited in some accessibility / usability pass over the redesigned code.